package.json

```diff
@@ -14,8 +14,10 @@
   "scripts": {
     "build": "webpack -p --mode production",
     "build-dev": "webpack --mode development",
+    "build-test": "webpack --config ./test/webpack.config.js --mode development",
```

Comment:
we should make this "build:test" in office-addin-taskpane separately, and include that change with the next round of merging changes. 

Likewise, "dev-server:test".